### DISCUSSION TITLE
Changing screen resolution

### DISCUSSION THREAD

**Discussion by jstavats** [fayaaz/mixxx-pi-gen#47] [General]
Hi, is there an easy method in this to change the screen resolution?  mine comes up at 4k by default and the normal config.txt settings for display resolution only seem to apply until the GUI starts.  I want to be able to force it to 720p. Thanks,

**Comment by fayaaz** [comment]
I believe arandr is available on this image (if not `sudo apt-get update && sudo apt-get install arandr`)

Steps to change resolution:
- Hit Win + Enter to get a terminal
- Run `sudo arandr` in terminal
- Set the resolution and save the .sh file (iirc that should save it somewhere where it runs on startup/graphical target)
